content/applications/sales/point_of_sale/restaurant/tips.rst

```diff
@@ -1,31 +1,88 @@
 
-Configure Tipping
-=================
+.. _configuration:
 
-To activate the *Tips* feature, go to :menuselection:`Point of Sale
---> Configuration --> Point of sale` and select your PoS.
+Configuration
+=============
 
-Under the Bills & Receipts category, you will find *Tips*. Select it
-and create a *Tip Product* such as *Tips* in this case.
+To allow tipping in your POS, activate the :guilabel:`Tips` feature in :menuselection:`Point of Sale
+--> Configuration --> Settings`. At the top of the page, select the POS in which you wish to allow
+**tipping**, scroll down to the :guilabel:`Payment` section and check :guilabel:`Tips`. Once
+enabled, add a :guilabel:`Tip Product` in the corresponding field, and save. The designated product
+will be used as a reference on customers' receipts.
 
-.. image:: tips/tips01.png
-    :align: center
+.. image:: tips/tips-setup.png
+   :align: center
+   :alt: enable tips in a POS
 
-Add Tips to the bill
-====================
+.. _tip-product:
 
-Once on the payment interface, you now have a new *Tip* button
+Tip products
+------------
 
-.. image:: tips/tips02.png
-    :align: center
+**Tip products** can be created on the spot. To do so, enter a product's name in the :ref:`Tip
+Product <configuration>` field and click :guilabel:`Create` or press **enter**. The product is
+automatically configured to be used as a tip at the payment screen.
 
-Add the tip your customer wants to leave and process to the payment.
+However, if you wish to be able to select the tip product in a POS session, you must activate the
+**Available in POS** setting. To do so, click :guilabel:`Create and edit...` instead of
+:guilabel:`Create` after you enter the product's name to open the product configuration form. Then,
+tick the :guilabel:`Available in POS` checkbox in the :guilabel:`Sales` tab and click
+:guilabel:`Save & Close`.
 
-.. image:: tips/tips03.png
-    :align: center
+.. note::
+   - When you create a product to use as a tip, leave the **product type** as :guilabel:`Consumable`
+     to avoid unnecessary inventory movements.
+   - You can only select one tip product per POS, but you can choose a different one for each.
+
+Add tips
+========
+
+To add tips to an order, :doc:`access the payment screen <../overview/getting_started>` and click
+:guilabel:`♥ Tip`. Then, enter the tipping amount, click :guilabel:`Confirm` to validate, and
+process the payment.
+
+.. image:: tips/add-tip.png
+   :align: center
+   :alt: tip popup window
+
+Alternatively, you can select the :ref:`tip product <tip-product>` on the POS interface to add it to
+the cart. When selected, the product is automatically set as a tip, and its default value equals its
+**Sales Price**.
+
+Tip using an Adyen terminal
+---------------------------
+
+If you use an :doc:`Adyen <../payment/adyen>` payment terminal and wish to enable **tips**
+using the terminal, check :guilabel:`Add tip through payment terminal (Adyen)` when activating the
```

Comment:
nitpicking, but it can happen at different times and not only when the admins activates the tip setting.